hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -2436,10 +2442,8 @@ public boolean reportRegionStateTransition(final RegionStateTransitionContext co
     final ReportRegionStateTransitionRequest request =
         createReportRegionStateTransitionRequest(context);
 
-    // Time to pause if master says 'please hold'. Make configurable if needed.
-    final long initPauseTime = 1000;
     int tries = 0;
-    long pauseTime;
+    long pauseTime = this.retryPauseTime;
```

Comment:
Is this a retry pause time or init pause time? If it is retry in the config also add that 'retry' in the name of the config.

Reply:
It is initial pause time. I will change the var name to "initPauseTime" to avoid confusion, will it be fine?